cfpb/cfgov-refresh
# Housing counselor: cache the map marker DOM lookups

A DOM lookup was happening every time a map marker was clicked. This caches the lookups so they only happen on the first time the item is clicked.

## Changes

- Cache result list item DOM references when the map markers are clicked.

## Testing

1. `gulp clean && gulp build`
2. Visit http://localhost:8000/find-a-housing-counselor/
3. Enter a ZIP and find results.
4. Click on a map marker to scroll to the results list item.
5. Click on the same map marker again.
6. Click on a different map marker.
7. Search for a different ZIP code.
8. Click on a marker and see that it scrolls to the results list item.

If you want, you can add a `console.log` where the DOM lookup happens and see that it will only be called once per ZIP code query.
